src/CircFit.jl

```diff
@@ -1,7 +1,36 @@
 module CircFit
 import LsqFit: lmfit
```

Comment:
Maybe use LsqFit for testing (i.e. add it as a test dependency), but there seems to be no need to have it as a module dependency now that you have the analytical solution.  The analytical solution should be faster, more accurate, and more reliable.

Reply:
It seems the analytic version works fine, so I removed the versions containing  lmfit